test/sqs_lambda_to_batch_fargate/lambda/export_logs.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+import os
+import time
+
+import boto3
+
+s3_client = boto3.client("s3")
+s3_resource = boto3.resource("s3")
+logs_client = boto3.client("logs")
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def handler(event, context):
+    logger.info(f"INCOMING EVENT: {event}")
+    if "Batch Job State Change" in event["detail-type"]:
+        if (
+            "FAILED" in event["detail"]["status"]
+            or "SUCCEEDED" in event["detail"]["status"]
+        ):
+            logger.info(f"BUCKET_NAME: {os.environ['BUCKET_NAME']}")
+            try:
+                log_group_name = "/aws/batch/job"
+
+                # Get most recent stream
```

Comment:
Extract this body to a stand-alone function because we might at some point try to get around to unit testing it